ludwig/datasets/configs/sarcastic_headlines.yaml

```
@@ -0,0 +1,7 @@
+version: 1.0
+name: sarcastic_headlines
+train_filenames: Sarcasm_Headlines_Dataset.json
+archive_filenames: news-headlines-dataset-for-sarcasm-detection.zip
```

Adds shasum to verify dataset download

```suggestion
archive_filenames: news-headlines-dataset-for-sarcasm-detection.zip
sha256:
  news-headlines-dataset-for-sarcasm-detection.zip: 3728f0fbce563536c3c67ab92e343e3ebcdc5cf1feaf4980c3abd4e54109eb51
```